apps/web/components/export/PreviewResult.tsx

```diff
@@ -10,22 +10,7 @@ export function PreviewResult(props: PreviewQuery) {
   const {headings, rows, totalCount} = data
 
   if (isInitial) {
-    return isFetching ? (
-      <LoadingPreviewResult />
-    ) : (
-      <EmptyPreviewResult
-        title="Select a table to export from!"
-        action={
-          <p className="max-w-[14rem] text-center">
-            <Link
-              className="text-venice-green hover:text-venice-green-darkened"
-              href="/connections">
-              Or, connect more financial institutions.
-            </Link>
-          </p>
-        }
-      />
-    )
```

Comment:
Sneak in this clean-up. Since we select a table by default and kicks off the query on page mount, this will no longer hit/needed.